### ISSUE TITLE
Could not resolve when running `tns test android --bundle`

### ISSUE THREAD

**Issue by DeadEnglish** [NativeScript/nativescript-cli#4580]
**Environment**
tns info result:
```
✔ Getting NativeScript components versions information...
✔ Component nativescript has 5.3.4 version and is up to date.
✔ Component tns-core-modules has 5.3.1 version and is up to date.
✔ Component tns-android has 5.3.1 version and is up to date.
✔ Component tns-ios has 5.3.1 version and is up to date.
```

**Describe the bug**
When running `tns test android --bundle`, component paths cannot be resolved and produces the following error:
```
Error: Could not resolve /main-page.component.html. Looked for: /main-page.component.html.
```

**To Reproduce**
* Download sample project
* run `tns test --bundle`

**Expected behavior**
tests should run and pass as expected (much like the behaviour when omitting `--bundle`)

**Sample project**
https://github.com/DeadEnglish/tns-test-bundle

**Additional context**
Anything I can do to help just let me know, keen to get this working 😃

**Comment by Fatme** [comment]
@DeadEnglish,

Thank you for sending us a sample project!

I've noticed that there isn't `karma.conf.js` file in  your project. Maybe you just missed to commit it, as I see you have `karma` as devDependency [here](https://github.com/DeadEnglish/tns-test-bundle/blob/d3f23817c9ab0e6477de1fd99fad5fa23e4f8ed2/package.json#L31).  Anyway, you need to execute `tns test init` and it should resolve the problem with missing `karma.conf.js` file.

Your project is actually an application created with [typescript template](https://github.com/DeadEnglish/tns-test-bundle/blob/master/app/app.ts#L9) but you're trying to add an [angular component](https://github.com/DeadEnglish/tns-test-bundle/blob/master/app/main-page.component.ts#L4) in it. This is not a supported scenario. If you want to use angular, you need to create the application from `angular` template. More info can be found [here](https://docs.nativescript.org/tooling/docs-cli/project/creation/create).

You're trying to write the tests using [testBed](https://github.com/DeadEnglish/tns-test-bundle/blob/master/app/tests/example.ts#L3) but it is applicable only for `angular` projects. It cannot be used in typescript projects. This is the reason to see `Error: Could not resolve /main-page.component.html. Looked for: /main-page.component.html.`.

**Comment by DeadEnglish** [comment]
HI @Fatme,

Sorry about that, the default .gitignore ignores all `.js` files except the `webpack.config.js`. I have rectified this and the project should now include the karma config.

This was technically just a playground app as our main app is an angular repo using typescript and have had multiple `tns create` to try and get this working aha. I have recreated the example using the angular base and it's working perfectly. Thanks for the help 😄.
